pkg/policy/l4.go

```diff
@@ -438,6 +438,16 @@ func (l4 *L4Filter) IsRedirect() bool {
 	return l4.L7Parser != ParserTypeNone
 }
 
+// IsEnvoyRedirect returns true if the L4 filter contains a port redirected to Envoy
```

Comment:
Add unit tests for the functions added in this file.

Reply:
Added asserts with the new functions to existing tests.

Reply:
Added to existing functions